### PR TITLE
Ft/nav bar

### PR DESCRIPTION
**Pull request by beylar**
Sticky navbar and implemented changes

### CODE REVIEW

**Review by Chartine02 — APPROVED** [review]

**Review by gloriaumutoni — APPROVED** [review]

File: src/sections/header.html [hunk 1]
```diff
@@ -1,4 +1,14 @@
-<div>
-    <h1 class="text-blue-800">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="flex justify-between px-7 py-3">
+    <div class="flex flex-col text-2xl">
+        <p>DE</p>
+        <p>EN</p>
+
```
Comment (NoellaPromise) [inline src/sections/header.html]: You may remove this space for the cleanness of the codes

File: index.html [hunk 2]
```diff
@@ -25,10 +25,12 @@
...
 </script>
-<body>
-    <div id="header"></div>
+<body class="relative">
+    <div id="header" class="sticky top-0 z-50"></div>
     <div id="hero"></div>
     <div id="contact"></div>
     <div id="footer"></div>
+
+
```
Comment (Kingcedru) [inline index.html]: You can remove these empty lines.

File: src/sections/header.html (hunk 1 above)
Comment (Kingcedru) [inline src/sections/header.html]: Remove any necessary empty lines

File: tailwind.config.js [hunk 3]
```diff
@@ -16,7 +16,9 @@ module.exports = {
     },
     colors: {
       "custom-green": "#00672E",
+      "custom-bgcolor": "#EFECEA"
     },
+
```
Comment (Kingcedru) [inline tailwind.config.js]: The same here empty lines.

**Review by Kingcedru — CHANGES_REQUESTED** [review]
You can remove necessary spaces for this pull request

**Review by NoellaPromise — APPROVED** [review]

**Review by Kingcedru — APPROVED** [review]

**Pull request merged**